### PR TITLE
Update CommunicationTurn swagger and add CommunicationIdentity swagger to preview folder

### PR DESCRIPTION
**Pull request by apattath**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

Following changes are made in this PR:
Updates to CommunicationTurn swagger (Note, this Turn swagger is not yet released to public):
1. To use shared types for CommunicationErrorResponse from stable common.json
    - Note: this causes a breaking change, for which there's already a bug filed on the service side to return innererror instead of innerError (https://skype.visualstudio.com/SPOOL/_workitems/edit/2370190). 
2. To rename Turn types by adding prefix Communication to be consistent with other models in the Communication.Identity package 

Update to CommunicationIdentity swagger: 
This change is needed since CommunicationIdentity and CommunicationTurn both go into the same SDK package and need to align on version (otherwise the generated SDK hardcodes the newer version).
1. Create a copy of current stable CommunicationIdentity.json (and examples) under a sub-folder under data-plane\Microsoft.CommunicationServicesIdentity\preview\2021-02-22-preview1\CommunicationIdentity.json. 
2. Update the version of CommunicationIdentity swagger under this folder to 2021-02-22-preview1 to match with CommunicationTurn version (as both are going into the same SDK package)

Update to Readme.md
1. Add new version of CommunicationIdentity.json to the input file flag.

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? By next week
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by apattath** [comment]
@RezaJooyandeh @DominikMe @ajpeacock0 @alexandra142 - tagging all of you for review as well.

**Comment by apattath** [comment]
I believe the breaking change is from the change made by @DominikMe in his earlier PR to use the common type for CommunicationErrorResponse.

**Review by tjprescott — APPROVED** [review]
@johanste could you provide the approval?

Justification:

None of the SDK is released.
The breaking change is about renaming a property from "innerError" to "innererror" (the new name follows the API guideline ref).
The service has not yet been picked by customers, and the innerError property is not populated at the moment.

Same rationale as in #12799.

File: specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-02-22-preview1/CommunicationIdentity.json [hunk 1]
```diff
@@ -0,0 +1,313 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "version": "2021-02-22-preview1"
```
Comment (alexandra142) [inline specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-02-22-preview1/CommunicationIdentity.json:6]: Why it asses postfix preview1? 
Internally in code I can see this controller under V4 folder. 
As V2 has postfix preview2 .. then I suggest Use postfix preview4 ? Or what is your proposal?

Comment (apattath) [reply specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2021-02-22-preview1/CommunicationIdentity.json:6]: I need the version here to match the one in the TURN swagger to be able to generate the SDK correctly. TURN swagger has preview1 suffix as it's the first preview for TURN service endpoint. Hence I used preview1 suffix here.

**Pull request merged**